libretro/jni/Application.mk

```diff
@@ -1 +1,2 @@
+APP_STL := c++_static
```

Comment:
There's no need for this, SameBoy does not use C++.

Reply:
``` suggestion
```